.github/workflows/check-submodule-pointers.yml

```diff
@@ -0,0 +1,25 @@
+name: Check submodule pointers
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check-submodules:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Compare submodule pointers to lgtm.com branch
+        env: 
+          GITHUB_TOKEN: ${{ github.token }}
+        run: |
+            if [ $(git rev-parse @:./ql) != $(gh api repos/github/codeql/git/ref/heads/lgtm.com --jq '.object.sha') ] || \
+               [ $(git rev-parse @:./codeql-go) != $(gh api repos/github/codeql-go/git/ref/heads/lgtm.com --jq '.object.sha') ]
+            then
+              echo "Submodules are out-of-date. Please update submodule pointers to the latest lgtm.com branch."
+              exit 1
```

Comment:
It would be really neat if you could create a PR for the change right here. But that could happen later in a future PR.

See here https://github.com/github/vscode-codeql/blob/31b445c8d2db8e0d49409d0333f9c40f1347325a/.github/workflows/release.yml#L122-L132 for an idea on how to do this.